test/unit/UnitTests.html

```diff
@@ -19,6 +21,14 @@
 				}
 			}
 		</script>
+		<script>
+			window.QUnit.on( 'runEnd', ( runEnd ) => {
```

Comment:
I tried to execute this in the puppeteer script inside a separate `page.evaluate` but for some reason it did not run. I didn't investigate deeper since here it executes fine.